code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -150,6 +150,18 @@
 	O.update_icon()
 	return
 
+
+/obj/item/weapon/storage/box/syndie_kit/imp_mindslave
+	name = "Mindslave Implant (with injector)"
+
+/obj/item/weapon/storage/box/syndie_kit/imp_mindslave/New()
+	var/obj/item/weapon/implanter/O = new(src)
+	O.imp = new /obj/item/weapon/implant/mindslave(O)
+	O.update_icon()
+	..()
```

Comment:
..() should be called at the start of the proc,you need to create the box first then fill it